integ/src/eks_provider.rs

```diff
@@ -310,24 +387,11 @@ async fn instance_profile(
         .send()
         .await
         .context("Unable to list instance profiles")?;
-    let eksctl_prefix = format!("eksctl-{}", cluster_name);
     list_result
         .instance_profiles
         .as_ref()
         .context("No instance profiles found")?
         .iter()
-        .filter(|x| {
```

Comment:
Is this from previous PR for the sg group?

Reply:
It's about finding instance profile not related to SG. The logic here is pretty strict and not suitable for different situations, so I think we don't need this anymore. Actually I have this code change in my NodeGroup PR, but I think it might be worth to add here.